src/main/java/se/diabol/jenkins/pipeline/DeliveryPipelineView.java

```diff
@@ -117,13 +112,20 @@
 
     private transient String error;
 
+    private static Random generator = new Random();
+    private String id = null;
+    private String viewUrl = null;
+    private boolean isPortletView = false;
```

Comment:
@tommysdk 
I was not able to move the configuration to another class to isolate the portlet configuration.
Will submit the rebased branch code in the evening.